src/main/java/ch/njol/skript/expressions/ExprOps.java

```diff
@@ -71,6 +87,8 @@ public Class<?>[] acceptChange(ChangeMode mode) {
 
 	@Override
 	public void change(Event event, @Nullable Object[] delta, ChangeMode mode) {
+		if (nonOps)
```

Comment:
This can be also added to acceptChange method to return null which will throw an error when someone tries to change non-ops